parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -336,6 +336,28 @@ else
             systemctl restart --no-block apt-daily.service
         fi
         aptmarkWALinuxAgent unhold &
+    elif [[ $OS == $MARINER_OS_NAME ]]; then
+        {{- if EnableUnattendedUpgrade }}
```

Comment:
please avoid using go templates anywhere other than `cse_cmd.sh`, the goal is to remove them entirely.

can use `        if [ "${ENABLE_UNATTENDED_UPGRADES}" == "true" ]; then` here which is already plumbed for ubuntu if you rebase